ros2doctor/README.md

```diff
@@ -0,0 +1,27 @@
+# ros2doctor
+
+This folder contains the source code of ros2doctor, part of the ROS 2 command line interface tools included with a standard install of any ROS 2 distro. ros2doctor command line tool is similar to roswtf in ROS system. It will examine your ROS 2 setup, such as distribution and platform, environment variables, network etc. and look for potential issues.
```

Comment:
```suggestion
This folder contains the source code of ros2doctor, part of the ROS 2 command line interface tools included with a standard install of any ROS 2 distro. 
ros2doctor command line tool is similar to roswtf in ROS system. 
It will examine your ROS 2 setup, such as distribution and platform, environment variables, network etc. and look for potential issues.
```
Each sentence has to be on a new line